src/Diagnostics.ModelsAndUtils/Models/Response.cs

```diff
@@ -38,6 +38,11 @@ public class Response
         /// </summary>
         public List<AzureSupportCenterInsight> AscInsights { get; set; }
 
+        /// <summary>
+        /// List of detector forms - DO NOT ADD TO THIS DIRECTLY
+        /// </summary>
+        public List<Form> DetectorForms { get; set; }
```

Comment:
Why do we need a different member object for Forms? This should be part of DataSet, no?

Reply:
So the main purpose of having a different member DetectorForms is to validate all the forms per detector in the response sent back to the UI (Refer to ValidateForms method in DiagnosticControllerBase.cs). Forms are added part of the DataSet by using res.AddForm or res.AddForms - here we can only validate individual forms but not all the forms per detector. So by having a different member DetectorForms - we can perform validations across Forms such as each form should have unique id